docs/config-file-curations-yml.md

```diff
@@ -48,7 +48,7 @@ location of source artifacts.
 
 Hint: If the `concluded_license` _and_ the `authors` are curated, this package will be skipped during the `scan` step,
 as no more information from the scanner is required. This requires the `skipConcluded` scanner option to be enabled in
-the [ort.conf](https://github.com/oss-review-toolkit/ort/blob/main/model/src/main/resources/reference.yml).
+the [config.yml](https://github.com/oss-review-toolkit/ort/blob/main/model/src/main/resources/reference.yml).
```

Comment:
Only semi-related: We should probably say "see `reference.yml`" here like in other similar places and not directly link `config.yml` to `reference.yml`, which might be confusing.

Or, thinking about it, do the exact opposite and always link to `config.yml` to `reference.yml` in the other places; there might not be much of a confusion after all, as there *is* no `config.yml` to link to.

Reply:
I have added a separate commit that changes those links to point to the README section about the config file instead, because that makes more sense to me.